2020/7xxx/CVE-2020-7331.json

```diff
@@ -13,28 +13,33 @@
                     "product": {
                         "product_data": [
                             {
-                                "product_name": "Endpoint Security for Windows",
+                                "product_name": "McAfee Endpoint Security (ENS) ",
                                 "version": {
                                     "version_data": [
                                         {
                                             "version_affected": "<=",
                                             "version_name": "10.7.x",
                                             "version_value": "10.7.0 September 2020 Update"
+                                        },
+                                        {
+                                            "version_affected": "<=",
```

Comment:
If version <= 10.6.1 September 2020 Update are affected then they should also be included in the description